lib/flux/app-state.ts

```diff
@@ -74,7 +73,7 @@ export const actionMap = new ActionMap({
 
       return update(state, {
         showNavigation: { $set: true },
-        showNoteInfo: { $set: false },
+        //showNoteInfo: { $set: false }, // @todo?
```

Comment:
as we'll capture this in the reducer when listening for `App.toggleNavigation` we can go ahead and remove it here